src/Table/Table.tsx

```diff
@@ -53,6 +53,8 @@ export interface ActionLinkItem {
 
 export interface PrimaryActionButton {
     label: string;
+    buttonTheme?: "link" | "outline-primary" | "secondary" | "ghost-dark" | "ghost-light" | "danger" | "primary";
+    buttonSize?: "btn-lg" | "btn-md" | "btn-sm";
```

Comment:
I guess you have to remove the `btn-` here or don't append it in the className

Reply:
Oh shit! I guess this is why unit test is important.